src/ai/backend/test/tester/exporter.py

```diff
@@ -43,64 +49,118 @@ async def export_stage_exception(self, stage: str, exception: BaseException) ->
         raise NotImplementedError("Subclasses must implement this method.")
 
     @abstractmethod
-    async def export_exception(self, spec_name: str, exception: BaseException) -> None:
+    async def export_exception(self, exception: BaseException) -> None:
         """
         Export the exception that occurred during the test run.
         """
         raise NotImplementedError("Subclasses must implement this method.")
 
 
-class PrintExporter(TestExporter):
+_DEFAULT_OUTPUT_DIRECTORY = Path("test_output")
+
+
+@dataclass
+class ErrorOutput:
+    test_id: str
+    spec_name: str
+    sub_name: Optional[str]
+    stage: str
+    exception_name: str
+    traceback: str
+
+
+class DefaultExporter(TestExporter):
     _sub_name: Optional[str]
     _started: float
+    _last_stage: Optional[str]
     _last_stage_done: float
+    _output_directory: Path
 
     def __init__(self, sub_name: Optional[str], started: float, last_stage_done: float) -> None:
         self._sub_name = sub_name
         self._started = started
+        self._last_stage = None
         self._last_stage_done = last_stage_done
+        self._output_directory = _DEFAULT_OUTPUT_DIRECTORY
```

Comment:
It seems that `_output_directory` is always using the default value.

Reply:
Yes. I'm just leaving it as the default value and I'm not planning on considering the output yet.
The part I'm worried about right now is that this variable is only used in default exporter, so its injecting with context is difficult to handle in common.